lib/3_add_up_numbers.php

```diff
@@ -7,9 +7,34 @@
 // Run `bin/check` from the command line to execute the automated tests.
 
 function addUpNumbers($arr) {
-  // Your code goes here.
+  $total = 0;
+  for ($i = 0; $i < count($arr); $i++) {
+    $total += $arr[$i];
+  }
+  return $total;
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
 
+echo "\nTesting that the array [1, 2, 3] adds up to 6.\n";
+if (addUpNumbers(array(1, 2, 3)) == 6) {
+  echo "Success!";
+}
+else{
+  echo "Failed!";
+}
+echo "\n-------------------------------------------------------------------\n";
+
+echo "\nTesting that the array [3, 2] adds up to 5.\n";
+if (addUpNumbers(array(3, 2)) == 5)  {
```

Comment:
Thanks Susan–you’re right that the instructions could be clearer. We’ll definitely make that improvement for the next class.

Of course, it’s still true that it could be helpful to have some additional test coverage (assuming the spec was updated to allow for non-positive or non-integer inputs to the function). If _only_ for the sake of getting more practice in thinking about test coverage, I suggest adding one or two more tests that cover some non-obvious or alternative situations.

I can see you’re very comfortable with the material so far, so I appreciate your hard work and patience as we ramp things up! Good work so far, and please feel free to reach out to me or Sumeet if you want extra work to keep yourself challenged. (And of course, expect us to continue to push you on assignments–even if that means going slightly outside a conservative interpretation of the assignment!)